src/main/java/thrift/model/transaction/Transaction.java

```diff
@@ -42,4 +42,21 @@
      * @return Set&lt;Tag&gt; belonging to this Transaction object.
      */
     public abstract Set<Tag> getTags();
+
+    /**
+     * Gets this Transaction object's {@code}isUpdated{@code} boolean value
+     *
+     * @return This Transaction object's isUpdated boolean value
+     */
+    public abstract boolean getIsJustUpdated();
+
+    /**
+     * Sets this Transaction object's {@code}isUpdated{@code} boolean value to {@code}true{@code}
+     */
+    public abstract void setIsJustUpdated();
```

Comment:
Same as above. Is it possible to have just one method `setIsJustUpdated(boolean isJustUpdated)` instead of 2 different methods?